CONTRIBUTING.md

```diff
@@ -8,26 +8,16 @@ Please respect the license of any existing files (specified at the top) and if a
 Please note that this project is released under the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md).
 By participating in this project you agree to abide by its terms.
 
-## Development and Testing
+## Reporting problems
 
-When developing this project, the following tools are necessary:
+### Before reporting issues
 
-* Git
-* One of either:
-  * GCC or Clang (Clang is not strictly officially supported)
-  * [`arm-none-eabi-gcc`](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/downloads) (from ARM, self-built and distro-supplied is currently broken)
-* GNU Make or compatible `make` tool
```

Comment:
I think keeping the section but linking it to the README would be best, I removed it because it would have to be duplicated here

Is the README not the more typical point of entry? or is this about pointing from the PR template?

Reply:
It's about it pointing from the PR template, yeah - hence too being fine saying "see this passage in the README" as long as that link's there.